README.md

```diff
@@ -1,11 +1,17 @@
 # APM PARSER
 
+## Cli usage
+```typescript
+yarn cli -u <username> -p <password> -c "https://apm-7-17.es.us-central1.gcp.cloud.es.io:9243" -s 2022-04-12T00:01:00.000Z -j local-90a41a83-90da-487b-a6be-4713a9ad18d8 -n 'My cool journey'
```

Comment:
optional nit: according to [this code](https://github.com/elastic/apm-parser/pull/4/files#diff-fa8d4e24d8399e8350f1c8bad05df53e8032ea995835bf911507015e2db61cddR24-R26) `-s` is `demandOption: true` option. I thought we decided to make it optional.